src/components/Navbar/AdminNavbar.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
navbars should be sticky to the top of the page, and span the entire width of the page. reference the figma as an example